src/components/ZeroState/ZeroState.tsx

```diff
@@ -1,27 +1,68 @@
 import { Suspense } from 'react';
-import { Bullseye, Button, Grid, Spinner } from '@patternfly/react-core';
+import {
+  Bullseye,
+  Button,
+  Card,
+  CardBody,
+  CardTitle,
+  Flex,
+  FlexItem,
+  Grid,
+  PageSection,
+  Spinner,
+  Text,
+  TextContent,
+  Title,
+} from '@patternfly/react-core';
 import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
 import ErrorState from '@redhat-cloud-services/frontend-components/ErrorState';
 import { createUseStyles } from 'react-jss';
 
 import { useAppContext } from 'middleware/AppContext';
+import { useHref, useNavigate } from 'react-router-dom';
+import { POPULAR_REPOSITORIES_ROUTE, REPOSITORIES_ROUTE } from 'Routes/constants';
 
 const useStyles = createUseStyles({
   contentZerostate: {
     minHeight: '100%',
     '& .bannerBefore': { maxHeight: '320px!important' },
     '& .bannerRight': { justifyContent: 'space-evenly!important' },
   },
+  textContent: {
+    minHeight: '40px',
+  },
+  removeBottomPadding: {
+    paddingBottom: '0',
+  },
 });
 
 export const ZeroState = () => {
   const classes = useStyles();
+  const navigate = useNavigate();
   const { setZeroState } = useAppContext();
+  const path = useHref('content');
+  const pathname = path.split('content')[0] + 'content';
 
-  const handleClick = () => {
+  const handleMainButtonClick = () => {
     setZeroState(false);
   };
 
+  const repoList = [
+    {
+      title: 'Red Hat repositories',
+      description: 'Browse available Red Hat Repositories to create RHEL images.',
+      onClick: () => {
+        setZeroState(false);
+        navigate(`${pathname}/${REPOSITORIES_ROUTE}?origin=red_hat`);
+      },
+    },
+    {
+      title: 'Popular repositories',
+      description: 'Add popular repositories with a single click.',
+      onClick: () => navigate(`${pathname}/${REPOSITORIES_ROUTE}/${POPULAR_REPOSITORIES_ROUTE}`),
```

Comment:
oh maybe the behavior i'm seeing in my previous comment is because we need to `setZeroState(false)` here?

Reply:
Yes! good catch! thank you!